index.html

```diff
@@ -424,6 +444,84 @@ <h2 class="experience-text">Skills</h2>
           </li>
         </ul>
       </section>
+
+      <!-- Contact 4/4 - Layout -->
+
+      <section class="l-contact">
+        <!-- Background Images -->
+
+        <div class="e-12"></div>
+        <div class="e-3"></div>
+        <div class="e-5"></div>
+        <div class="r-7"></div>
+        <div class="r-7 r-7-b"></div>
+
+        <!-- Block 1/3 - Text -->
+
+        <div class="l-contact__text">
+          <h2 class="text-title">Contact me</h2>
+          <p class="text-parraf">
+            If you have an application you are interested in developing, a
+            feature that you need built or a project that needs coding. I’d love
+            to help with it
+          </p>
+        </div>
+
+        <!-- Block 2/3 - Form -->
+
+        <form action="https://formspree.io/f/mwkaodba" method="post">
+          <ul>
+            <li>
+              <input
+                type="text"
+                class="input"
+                name="name"
+                id="full-name"
+                maxlength="30"
+                placeholder="Write your name here"
+                required
+              />
+            </li>
+            <li>
+              <input
+                type="email"
+                class="input"
+                name="_replyto"
+                id="email-address"
+                placeholder="Write your email here"
+                required
+              />
+            </li>
+            <li>
+              <textarea
+                class="input-message"
+                name="message"
+                id="message"
+                maxlength="500"
+                placeholder="Write your message here"
+                required
+              ></textarea>
+            </li>
+            <li>
+              <button
+                type="submit"
+                id="submit"
+                class="c-b-normal-button"
+                aria-disabled="true"
+                aria-describedby="full-name email-address message"
+              >
+                Get In Touch
+              </button>
```

Comment:
- [ ] Please make sure to follow the design template.

_Look at the comment above about the given design._